tests/tests.cc

```diff
@@ -623,7 +621,8 @@ class Tests {
       }
     }};
 
-    usleep(10 * 1000);  // sleep for 10ms.
+    std::this_thread::sleep_for(
+        std::chrono::milliseconds(10));  // sleep for 10ms.
```

Comment:
```suggestion
    std::this_thread::sleep_for(std::chrono::milliseconds(10));  // sleep for 10ms.
```

Reply:
clang-format won't let me do this, the line is too long. I could just remove the comment though since it's more obvious now.